crates/ibc/src/core/ics24_host/path.rs

```diff
@@ -44,6 +44,42 @@ pub enum Path {
     Upgrade(ClientUpgradePath),
 }
 
+#[cfg_attr(
+    feature = "parity-scale-codec",
+    derive(
+        parity_scale_codec::Encode,
+        parity_scale_codec::Decode,
+        scale_info::TypeInfo
+    )
+)]
+#[cfg_attr(
+    feature = "borsh",
+    derive(borsh::BorshSerialize, borsh::BorshDeserialize)
+)]
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
+#[derive(Clone, Debug, PartialEq, Eq, PartialOrd, Ord, Hash, Display)]
+#[display(fmt = "clients/{_0}/clientType")]
+pub struct ClientTypePath(pub ClientId);
+
+impl ClientTypePath {
+    pub fn new(client_id: &ClientId) -> ClientTypePath {
+        ClientTypePath(client_id.clone())
+    }
+}
```

Comment:
This `ClientTypePath` should be dropped. Check [here](https://github.com/cosmos/ibc-rs/pull/642) please.
Otherwise looks good to me :pray: